### DISCUSSION TITLE
How to delete shorts?

### DISCUSSION THREAD

**Discussion by GuitarFan** [uBlockOrigin/uAssets#18507] [Q&A] [answered]
Tell me please how to delete shorts from youtube trends on this page youtube.com/feed/trending ?
Standard tools of selecting don't work. I need to delete DIV which contains <a href="/shorts/*">

**Comment by MasterKia** [comment, marked as answer]
https://letsblock.it/filters/youtube-shorts

**Reply by GuitarFan** [reply]
Thank you. It works.
